lib/fauxpaas/release_builder.rb

```diff
@@ -37,16 +37,10 @@ def deploy_config(signature)
       end
     end
 
-    def extract_shared!(signature, shared_path)
-      fs.mkdir_p(shared_path)
-      signature.shared.each {|ref| add_reference(ref, shared_path) }
-      shared_path
-    end
-
-    def extract_unshared!(signature, unshared_path)
```

Comment:
Were these seriously just two functions that did the same thing but with different variable names? I like extract_ref a lot better.